test_collections/matter/scripts/OTBR/otbr_start.sh

```diff
@@ -1,24 +1,25 @@
 #! /usr/bin/env bash
-
- #
- # Copyright (c) 2023 Project CHIP Authors
- #
- # Licensed under the Apache License, Version 2.0 (the "License");
- # you may not use this file except in compliance with the License.
- # You may obtain a copy of the License at
- #
- # http://www.apache.org/licenses/LICENSE-2.0
- #
- # Unless required by applicable law or agreed to in writing, software
- # distributed under the License is distributed on an "AS IS" BASIS,
- # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- # See the License for the specific language governing permissions and
- # limitations under the License.
+#
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 ROOT_DIR=$(realpath $(dirname "$0")/../../../../..)
 TH_SCRIPTS_DIR="$ROOT_DIR/scripts"
 
 DEFAULT_OTBR_INTERFACE="eth0"
 BR_INTERFACE=${1:-$DEFAULT_OTBR_INTERFACE}
+BR_VARIANT="35" TODO: Receive the BR variant via shell script parameter
```

Comment:
```suggestion
# limitations under the License.
ROOT_DIR=$(realpath $(dirname "$0")/../../../../..)
TH_SCRIPTS_DIR="$ROOT_DIR/scripts"

BR_VARIANT="35"
if [ $# == 1 ]; then
   BR_VARIANT=$1
fi

DEFAULT_OTBR_INTERFACE="eth0"
BR_INTERFACE=${1:-$DEFAULT_OTBR_INTERFACE}
```

Reply:
This is not working. Don't know why yet

Reply:
Now I get it.
The first parameter `$1` it's already being configured to the `BR_INTERFACE` variable. It's just a different syntax.
Using a second parameter will require the user always give the interface (e.g. `otbr-start.sh eth0 45`) and It's a strange solution to me.

I'll see what I can do.

Reply:
Updated the script to accept up to two parameters, `-i` (for BR Interface) and `-v` (for BR Variant).
Please take a look!

Done in https://github.com/project-chip/certification-tool-backend/pull/150/commits/5f02ca8f871d73c1fb338aaadbdca742a9ffb9f5